components/builder-worker/src/config.rs

```diff
@@ -54,9 +53,7 @@ impl Default for Config {
         jobsrv.insert("heartbeat".to_string(), "5567".to_string());
         Config {
             auth_token: "".to_string(),
-            data_path: hab_core::fs::svc_data_path("hab-builder-worker")
-                .to_string_lossy()
-                .into_owned(),
+            data_path: "/tmp".to_string(),
```

Comment:
This is the default configuration for running the service by `cargo run`. It's set to `/tmp` (should probably be `std::env::temp_dir()` like the others) because you may not have the Habitat directory structure on your machine when running tests. This wasn't previously an issue because event logging is disabled by default. If anyone changed that, it would break the test suite.

I actually only made this change becuase `hab_core::fs::svc_data_path()` no longer exists. That's part of the `habitat_sup` module and not `hab_core`